src/universal/modules/email/components/NotificationSummary/NotificationSummaryEmail.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import makeAppLink from 'server/utils/makeAppLink';
+import appTheme from 'universal/styles/theme/appTheme';
+
+import Body from '../Body/Body';
+import Button from '../Button/Button';
+import Callout from '../Callout/Callout';
+import ContactUs from '../ContactUs/ContactUs';
+import Header from '../Header/Header';
+import Layout from '../Layout/Layout';
+
+export default function NotificationSummaryEmail() {
+  const notificationPageUrl = makeAppLink('me/notifications');
+  return (
+    <Layout>
+      <Header color={appTheme.palette.dark} />
+      <Body>
+        <p>Hi there, %recipient.name%!</p>
+        <p>You have received <span style={{fontWeight: 'bold'}}>%recipient.numNotifications% new notification(s)</span> in the last day.</p>
+        <Callout>
+          <Button url={notificationPageUrl}>See My Notifications</Button>
+        </Callout>
+      </Body>
+      <p>This is just a friendly, nudge! Your teammates need you!</p>
```

Comment:
i think we still need some wordsmithing here, but we can chat about that outside of GH

Reply:
For sure. I actually kicked this over to @ackernaut, who's taking a pass on the HTML email, which I'd say can include any appropriate wordsmithing. Happy to chat out of band on the figma mockups.